functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
@wsmelton I wonder if we should consider renaming `New-DbaAgentSchedule` then - if you create a schedule using that command, it comes back in the collection retrieved by this one. Having said that, the `New-DbaAgentSchedule` also seems to require and attach itself to a job even though it says it doesn't do the latter in the synopsis. I need the ability to create a shared schedule without referencing a job, so I might add some more stuff to this PR purely based on shared schedules